constance/backends/database/__init__.py

```diff
@@ -87,7 +86,7 @@ def set(self, key, value):
         if self._cache:
             self._cache.set(key, value)
 
-        updated_signal.send(sender='constance', key=key, value=value)
+        signals.config_updated.send(sender='constance', key=key, value=value)
```

Comment:
Passing in a `sender` parameter as a string doesn't make sense here, since it's used for [filtering down callbacks](https://docs.djangoproject.com/en/1.10/topics/signals/#connecting-to-specific-signals) in case a signal is used for different type of domain objects.

E.g. in the case of Django's model class signals like `post_save`, the signal is sent for no matter which model class. The difference between the call is the sender parameter.

Constance doesn't have multiple domain objects, so leaving the parameter to the default of `None` is better suited and even simplifies the developer facing API.

As a developer instead of using `updated_config.connect(sender='constance', config_changed_callback)` it's suffices to just use `updated_config.connect(config_changed_callback)`.

On the other hand, often signal callbacks are at risk of causing circular imports, and may not want to import the `constance.config` object in their module. In addition it makes sense for the developer who writes the signal callback to get something sent over that facilitated the config value change.

That's why I think it'd be best to pass in the `constance.config` object as the `sender` parameter, and document that that's what connected signal callbacks will receive as their first parameter. That way developers can created callback functions that are isolated and can act upon config changes by accessing the config object again as well.

IOW, this line should be `signals.config_updated.send(sender=config, key=key, value=value)`, where `config` is the lazy config objects `constance.config`.